nntrainer/src/databuffer_file.cpp

```diff
@@ -237,7 +236,6 @@ void DataBufferFromDataFile::updateData(BufferType type, int &status) {
       try {
         if (position > file_length || position > ULLONG_MAX) {
           ml_loge("Error: Cannot exceed max file size");
```

Comment:
Let's standardize not to use ml_log when throw since it will be handled :)
refering to: https://github.com/nnstreamer/nntrainer/pull/294#discussion_r453427550

Reply:
Done.